packages/svg-icons/package.json

```diff
@@ -24,7 +24,8 @@
         "./icons/*.svg": "./dist/icons/*.svg"
     },
     "scripts": {
-        "build": "tsx scripts/build.ts"
+        "build": "copyfiles --flat src/optimized-icons/* dist/icons",
```

Comment:
I'm super open to discussion here. 

In igloo-icons, everytime we build, we regenerate the components and optimize icons. It makes  sense because it's in a separate repo, and that's the intented purpose. 

However, in our mono repo, 90% of the time when we build, we don't have any changes to the icons. 

this is why the generate-icons command now handle the optimization + generation of react component separatly. The build steps is now a build with tsup for react component, and moving the pre-optimized icons from the src/optimized-icons to the dist folders for SVGs. 

Another advantage of this is we could run unit tests on optimized icons to ensure that the distributed package is following our standards as well and no errors occurred during the optimization.